### PR TITLE
Update Cobra.go to print version on stdout

### PR DESCRIPTION
**Pull request by jsenon**
Change Printf  in order to be able to parse result of command line istioctl version (ie. istioctl version | grep -I 0.5.1)

### CODE REVIEW

File: pkg/version/cobra.go [hunk 1]
```diff
@@ -27,9 +27,11 @@ func CobraCommand() *cobra.Command {
 		Short: "Prints out build version information",
 		Run: func(cmd *cobra.Command, args []string) {
 			if short {
-				cmd.Printf("%s\n", Info)
+// 				cmd.Printf("%s\n", Info)
+				fmt.Println(Info)
 			} else {
-				cmd.Printf("%s", Info.LongForm())
+// 				cmd.Printf("%s", Info.LongForm())
```
Comment (jwendell) [inline pkg/version/cobra.go]: Ditto

**Comment by geeknoid** [comment]
Would one of the solutions listed here be usable: https://stackoverflow.com/questions/2342826/how-to-pipe-stderr-and-not-stdout

I rather keep the code as-is since I think it's more inline with the spirit and expected use of Cobra.

**Comment by jsenon** [comment]
indeed we can do like this. But other command line like `kops version` or `docker version` could be used with `grep` without this trick.

**Comment by geeknoid** [comment]
Sorry, I didn't mean to close the PR.

**Comment by ldemailly** [comment]
thanks for the fix !

**Pull request merged**